### DISCUSSION TITLE
rates of all employees

### DISCUSSION THREAD

**Discussion by helgator** [kimai/kimai#4604] [Questions & Answers]
1. Where do I as the admin/employer see a table with all employees and their hourly rates?
2. How do I allow all employees to see their hourly rates?

**Comment by kevinpapst** [comment]
1. You don't.
2. See Roles & Permissions documentation, something like view `view_rate_own_timesheet` or `edit_rate_own_timesheet`
https://www.kimai.org/documentation/permissions.html

**Comment by helgator** [comment]
Thanks. Weird, so when I need to check the rates of my 18 employees, I have to edit every single one of them one by one just to see the rates?
That means I still have to run a separate software just to have an employee overview? Really weird. And yet another software where the employees can see their rates and earnings ... hmm

**Reply by kevinpapst** [reply]
Weird weird weird indeed. If you are a paying customer, be welcome to send me an email. 
If you want to become a paying customer, you are welcome to send me an email as well.
I am always open for improvements.

**Reply by helgator** [reply]
I did not know you maintain that thing all by yourself. Respect! 
Maybe I can buy the "Custom fields" plugin. Would that work for those two purposes?

**Comment by helgator** [comment]
Maybe I can buy the "Custom fields" plugin. Would that work for those two purposes?

**Reply by kevinpapst** [reply]
No. And I edited answer 1
